pkg/load/proxy.go

```diff
@@ -61,12 +61,18 @@ func RunProxyImage(ctx context.Context, dockerapi docker.APIClient, proxyImage s
 		return nil, err
 	}
 
+	attach, err := dockerapi.ContainerAttach(ctx, resp.ID, types.ContainerAttachOptions{Stdin: true, Stdout: true, Stderr: true, Logs: true, Stream: true})
+	if err != nil {
+		return nil, err
+	}
+
 	if err := dockerapi.ContainerStart(ctx, resp.ID, types.ContainerStartOptions{}); err != nil {
 		return nil, err
 	}
 
 	inspect, err := dockerapi.ContainerInspect(ctx, resp.ID)
 	if err != nil {
+		_ = StopProxyContainer(ctx, dockerapi, resp.ID)
```

Comment:
This was just a bonus cleanup step if there happens to be a container inspect issue.